test/suites/ACID.MATLAB.8.3.md5

```diff
@@ -40,7 +40,7 @@ legendsubplots : 528e57cc9c7858f9c941bd64733b2f12
 linesWithOutliers : 4976c3f7b1e8e8cd32f7e17e34fcc99f
 logbaseline : 59ff71a8a8125053e2ddac4dd3070e5a
```

Comment:
Newbie question, does somebody know the reason some tests have different hashes?